lib/matrix_check_sum.rb

```diff
@@ -3,8 +3,33 @@
 # of numbers in row i is the same as the sum of numbers in column i for i = 0 to row.length-1
 # If this is the case, return true. Otherwise, return false.
 
-# Time complexity: ?
-# Space complexity: ?
+# Time complexity: O(n^2), n^2 are the total number of elements in the matrix
```

Comment:
[nit] It's a little clearer to say that `n` is the size of a row or column of the square matrix.